_pages/en/open-source-codes.html

```diff
@@ -6,12 +6,19 @@
 ---
 
 <h1 property="name" id="wb-cont">Open Source Code</h1>
-<p>Canadian public administrations' open source projects and code repositories.</p>
-<p>See the list of <a href="./open-source-code-groups.html">Open Source Code Groups</a> to find more Canadian public administrations' open source projects and code repositories.</p>
-<p>See the <a href="{{ site.data.i18n.general.nav.doc.link[page.lang] }}">{{ site.data.i18n.general.nav.doc[page.lang] }}</a> and <a href="./open-source-code-schema.html">Open Source Code Metadata Schema</a> for the list of fields and how to add you data.</p>
+<p>
+  Open source code commonly refers to software that uses an open development process. This software is released under a license where
+  the copyright holder grants anyone permission to use, modify, change or redistribute the code for any purpose. This open source process
+  is an example of open collaboration where everyone benefits from the development of technology that solves real issues.
+</p>
+
+<h2 property="name">Software Projects</h2>
+<p>
+  Search open projects that are developed by different levels of Canadian public administrations (federal, provincial, municipal and aboriginal).
+  You may also add a new open source code to the following catalog, or update the information of a code that is already there, using the <a href="./open-source-code-form.html">Open Source Code Form</a>.<br>
```

Comment:
`<br>` tags are not optimal, especially for responsive. I would suggest using a different paragraph instead

Reply:
Outdated